execution-engine/src/hcall/common.rs

```diff
@@ -266,6 +142,15 @@ pub enum HostProvisioningError {
         display = "HostProvisioningError: Failed to sort the incoming data or incoming stream (this is a potential bug)."
     )]
     CannotSortDataOrStream,
+    #[error(
+        display = "HostProvisioningError: VFS Error {}.", _0
+    )]
+    VFSError(#[error(source)]VFSError),
+    //TODO: potential remove this 
```

Comment:
If this is still a relevant `TODO` please open an issue to track it.

Reply:
The TODO is legacy. Fixed.

Reply:
The TODO comment still seems to be there...